src/checks/series/flatline_check.rs

```diff
@@ -0,0 +1,107 @@
+use crate::{DataCache, Error, Flag};
+
+/// Timeseries QC test that checks for streaks of repeating values.
```

Comment:
```suggestion
/// Timeseries QC check that looks for streaks of repeating values.
```
Since we said we should use `check` instead of `test`, maybe you also want to leave out `QC`?